libraries/src/Application/CMSApplicationInterface.php

```diff
@@ -155,6 +155,15 @@ public function isCli();
 	 */
 	public function getIdentity();
 
+	/**
+	 * Gets the client id of the current running application.
+	 *
+	 * @return  integer  A client identifier.
+	 *
+	 * @since   __DEPLOY_VERSION_
```

Comment:
```suggestion
	 * @since   __DEPLOY_VERSION__
```
Repeat in the other files.